EsoTech.MessageQueue/AzureServiceBus/AzureServiceBusManager.cs

```diff
@@ -0,0 +1,148 @@
+﻿using Microsoft.Azure.ServiceBus;
+using Microsoft.Azure.ServiceBus.Management;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EsoTech.MessageQueue.AzureServiceBus
+{
+    public class AzureServiceBusManager
+    {
+        private readonly Dictionary<string, HashSet<string>> _topicks = new Dictionary<string, HashSet<string>>();
+        private readonly HashSet<string> _queues = new HashSet<string>();
```

Comment:
1. _topics, not _topicks.
2. Having these properties prevents users from calling UpdateSubscription multiple times for the same topic/subscription. and does not have MUCH added value because are called during initialization, right?